specification/voiceservices/data-plane/Microsoft.VoiceServices/stable/v1.0/swagger.json

```diff
@@ -0,0 +1,1052 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft.VoiceServices.ACG.ProvisioningPlatform.API",
+    "version": "v1.0"
+  },
+  "securityDefinitions": {
+    "OAuth2": {
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
```

Comment:
This is the flow that we need: https://learn.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow#get-a-token but it's swagger isn't compatible with tenant specific token URLs.

Please advise how we should handle this.

Reply:
Believe this is now resolved.